windows/vhba_hot_add_remove/create_new_file_read_write.yml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Set fact of the file path to do disk write test"
+  ansible.builtin.set_fact:
+    write_test_file_path_win: "{{ drive_letter_new }}:\\test_disk_write.txt"
+    test_line: "This is the test line to write to the file. "
+
+- include_tasks: ../utils/win_create_file.yml
+  vars:
+    new_empty_file_path: "{{ write_test_file_path_win }}"
+- include_tasks: ../utils/win_write_to_file.yml
+  vars:
+    write_file_path: "{{ write_test_file_path_win }}"
+    write_file_content: "{% for i in range(0, 20) %} '{{ test_line }}\n{{ test_line }}' {% endfor %}"
+
+- include_tasks: ../utils/win_check_file_exist.yml
+  vars:
+    win_check_file_exist_file: "{{ write_test_file_path_win }}"
+- name: "Check file created on new disk in guest OS"
+  ansible.builtin.assert:
+    that:
+      - "{{ win_check_file_exist_result }}"
+    success_msg: "New created file found in guest OS"
+    fail_msg: "New created file not found in guest OS: {{ write_test_file_path_win }}"
+
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "type {{ write_test_file_path_win }}"
+- name: "Save the result of getting new file content"
+  ansible.builtin.set_fact:
+    get_file_content_result1: "{{ win_powershell_cmd_output }}"
+- name: "Check file content"
+  ansible.builtin.assert:
+    that:
+      - get_file_content_result1.stdout is defined
+      - get_file_content_result1.stdout
+      - "test_line in get_file_content_result1.stdout"
+    fail_msg: "The file content is '{{ get_file_content_result1.stdout }}', which should contain: '{{ test_line }}'"
+
+# Check new created file still exist after guest OS restart
+- include_tasks: ../utils/win_shutdown_restart.yml
+  vars:
+    set_win_power_state: "restart"
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "type {{ write_test_file_path_win }}"
+    win_execute_cmd_ignore_error: true
+- name: "Save the result of getting new file content after guest restart"
+  ansible.builtin.set_fact:
+    get_file_content_result2: "{{ win_powershell_cmd_output }}"
+
```

Comment:
I suggest changing the tasks order as below after windows restart:
1. get disk online status firstly
2. Assert the disk is online when it doesn't meet lsilogicsas + hot-add + windows server conditions, otherwise fail the test case.
3. When it is offline and it meets lsilogicsas + hot-add + windows server conditions, treat it as known issue and online the disk
3. Get the file content on the disk
4. Check file content.

In such case, we can distinguish other kind of disk offline failure and file content failure.